one-pager/orchester/main.py

```
@@ -0,0 +1,102 @@
+            return {
+                "error": "Faltan los campos requeridos 'idea_description' o 'banner_prompt'"
+            }, 400
+
+        # --- 2. Llamada a la función que genera el texto ---
+        text_headers = {
+            "Authorization": f"Bearer {get_id_token(TEXT_FUNCTION_URL)}",
+            "Content-Type": "application/json"
+        }
+
+        print(f"Llamando a {TEXT_FUNCTION_URL}...")
+        text_response = requests.post(
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)